docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -4,11 +4,9 @@ title: Build a logs data source plugin
 
 # Build a logs data source plugin
 
-This guide explains how to build a logs data source plugin. 
+Data logs are defined as data stored over a period of time in different systems or environments for the purpose of tracking certain events.  
 
-Data logs are defined as data stored over a period of time in different systems or environments for the purpose of tracking a variety of events.  
-
-Grafana data source plugins support both metrics and log data. The steps to build a logs data source plugin are largely the same as for a metrics data source, but there are a few differences. 
+Grafana data source plugins support metrics , logs, and other data types. The steps to build a logs data source plugin are largely the same as for a metrics data source, but there are a few differences. 
```

Comment:
there is an extra space there: `metrics, logs, and other data types`

Reply:
fixed in latest commit